src/tasks/pulls.ts

```diff
@@ -270,6 +277,26 @@ export const pulls: PullSpec[] = [
   },
 ];
 
+if (bestCommaPull !== undefined) {
```

Comment:
Does this need to come as a separate addition? I think you can just include this once in the pulls array directly. 

Another note is that the pulls array is ordered by priority. Probably this saves about as many turns as GAP (if it is used) so you could put it around there.

Reply:
I couldn't figure out how to do it except as a separate addition. Because there's a chance of an undefined value we're a bit stuck.

Reply:
Actually, upon further thought `optional: true` already means that it should only be used if one exists in storage, not bought. So the following should actually do exactly what you want without needing `bestCommaPull`
```
pull: $items`aquaviolet jub-jub bird, charpuce jub-jub bird, crimsilion jub-jub bird, stomp box`,
optional: true
```

I guess there is a small detail that in `post` you have to figure out which one you have in order to eat it.

Reply:
That's easy peasy then; let's see if this works.